src/libraries/System.Net.WebSockets.Client/tests/WebSocketHelper.cs

```diff
@@ -97,25 +93,48 @@ public static Task<ClientWebSocket> GetConnectedWebSocket(
             Retry(output, async () =>
             {
                 var cws = new ClientWebSocket();
```

Comment:
```suggestion
                Retry(output, async () =>
                {
                    var cws = new ClientWebSocket();
```
I this is not changed in this PR, but the indent is wrong and I got confused here for while.